build.sbt

```diff
@@ -143,7 +148,14 @@ val verifyDeps = Seq(
   "org.ow2.asm" % "asm" sha1 "0da08b8cce7bbf903602a25a3a163ae252435795",
   "com.esotericsoftware" % "minlog" sha1 "ff07b5f1b01d2f92bb00a337f9a94873712f0827",
   "org.objenesis" % "objenesis" sha1 "272bab9a4e5994757044d1fc43ce480c8cb907a4",
-  "org.hamcrest" % "hamcrest-all" sha1 "63a21ebc981131004ad02e0434e799fd7f3a8d5a"
+  "org.hamcrest" % "hamcrest-all" sha1 "63a21ebc981131004ad02e0434e799fd7f3a8d5a",
+
+  // mallet
+  "com.github.scopt" % "scopt" sha1 "e078455e1a65597146f8608dab3247bf1eb92e6e",
+  "org.jline" % "jline" sha1 "dfb4e9e15e981634155ce063fa697b2b8964d507",
+  "org.scala-lang.modules" % "scala-parser-combinators" sha1 "3c1c5475ece77c41e18dd971f8f818c091e4961c",
+  // FIXME: WTF?
+  "com.google.j2objc" % "j2objc-annotations" sha1 "976d8d30bebc251db406f2bdb3eb01962b5685b3"
```

Comment:
I did a bit of searching. This is a dependency pulled in via guava `v22` (check image below) and is discussed [here](https://github.com/google/guava/issues/2824).


![selection_108](https://user-images.githubusercontent.com/36955/38602623-fae2ce4a-3d73-11e8-9be6-b57762f81f2c.png)

Reply:
What uses guava?

Reply:
Using `sbt` with `addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.0")`:
```
$ sbt whatDependsOn com.google.guava guava 22.0
[info] com.google.guava:guava:22.0
[info]   +-io.atomix:atomix-utils:2.1.0-beta1
[info]   | +-io.atomix:atomix-cluster:2.1.0-beta1
[info]   | [...]   
[info]   +-org.consensusresearch:scrypto_2.12:1.2.0-RC3 [S]
[info]   | [...]
[info]   +-org.scorexfoundation:iodb_2.12:0.3.0 [S]
[info]     
```

Reply:
@loverdos I removed the `j2objc-annotations` artifact from my local cache. After rebuilding, the checksum changed and it passed the verification. See also the updated PR description